server/coordinator/procedure/switch_leader.go

```diff
@@ -0,0 +1,38 @@
+package procedure
+
+import (
+	"context"
+)
+
+func NewSwitchLeaderProcedure() *SwitchLeaderProcedure {
+	return nil
+}
+
+func LoadSwitchLeaderProcedure(_ []byte) *SwitchLeaderProcedure {
```

Comment:
Procedure should not process `[]byte` directly, `List` func in `Storage` Process `[]byte` and return `*Meta`, so the parameter here should be `*Meta`